src/test/java/com/cappella/csv/CsvClientTests.java

```diff
@@ -0,0 +1,165 @@
+
+        }
+
+        @Test
+        void testMalformedTaskDueDateCsvFile() {
+                ClassLoader classloader = this.getClass().getClassLoader();
+                InputStream is = classloader.getResourceAsStream("malformedTaskDueDate.csv");
+                Problems problems = new Problems();
+                CsvClient csv = new CsvClient();
+                List<TaskData> grantTasks = csv.parseGrantCsvToTasks(is, problems);
+                Assertions.assertTrue(problems.getErrors().containsKey(Problems.ERROR_PARSING_TASK_DUE_DATE));
+        }
+
+        @Test
+        void testMalformedSubTaskDueDateCsvFile() {
+                ClassLoader classloader = this.getClass().getClassLoader();
+                InputStream is = classloader.getResourceAsStream("malformedSubTaskDueDate.csv");
+                Problems problems = new Problems();
+                CsvClient csv = new CsvClient();
+                List<TaskData> grantTasks = csv.parseGrantCsvToTasks(is, problems);
+                Assertions.assertTrue(problems.getErrors().containsKey(Problems.ERROR_PARSING_SUB_TASK_DUE_DATE));
+        }
+
+        @Test
+        void testSpecialCharactersCsvFile() {
+        }
+
+        @Test
+        void testOnlyNameCsvFile() {
+        }
+
+        @Test
+        void testSimpleCsvFile() {
+                ClassLoader classloader = this.getClass().getClassLoader();
+                InputStream is = classloader.getResourceAsStream("simple.csv");
+                Problems problems = new Problems();
+                CsvClient csv = new CsvClient();
+                List<TaskData> grantTasks = csv.parseGrantCsvToTasks(is, problems);
+                Assertions.assertTrue(problems.getErrors().isEmpty());
+                // should also assert that it successfully parsed the grant tasks expected
+        }
+
+        @Test
+        void testProductionCsvFile() {
+        }
+
+        @Test
+        void testParseGrantDueDateWrongFormat() {
+                Problems problems = new Problems();
+                CsvClient csv = new CsvClient();
+                String dateValue = "1/1/2001";
+                LocalDate dueDate = csv.parseGrantDueDate(dateValue, problems);
+                Assertions.assertNull(dueDate);
+                Assertions.assertTrue(problems.getErrors().containsKey(Problems.ERROR_PARSING_TASK_DUE_DATE));
+        }
+
+        @Test
+        void testParseGrantDueDate() {
+                Problems problems = new Problems();
+                CsvClient csv = new CsvClient();
+                String dateValue = "Jan 1, 2001";
+                DateTimeFormatter formatter = DateTimeFormatter.ofPattern("LLL d, yyyy", Locale.ENGLISH);
+                LocalDate date = LocalDate.parse(dateValue, formatter);
+                LocalDate dueDate = csv.parseGrantDueDate(dateValue, problems);
+                Assertions.assertTrue(date.equals(dueDate));
```

Comment:
Use Assertions.assertEquals instead of assertTrue.

Reply:
fixed in updated PR